src/LibLog/LibLog.cs

```diff
@@ -568,8 +625,10 @@ public bool Log(LogLevel logLevel, Func<string> messageFunc, Exception exception
                 return false;
             }
         }
+#endif
     }
 
+#if !LIBLOG_PROVIDERS_ONLY
```

Comment:
I think you'll still need this incase of mis-behaving loggers or something has broken. I know logging fwks are robust in this regard, but thinking about custom loggers...